support-frontend/.eslintrc.js

```diff
@@ -1,27 +1,34 @@
 module.exports = {
-  extends: [
-    '@guardian/eslint-config-typescript',
```

Comment:
Could we also remove this from our node modules

Reply:
As discussed it is still needed it's just moved lower in this file